src/main/java/org/folio/pv/service/ValidationRuleServiceImpl.java

```diff
@@ -51,20 +51,30 @@ public ValidationRule getValidationRuleById(String ruleId) {
 
   @Override
   public ValidationRuleCollection getValidationRules(Integer offset, Integer limit, String cql) {
-    var validationRuleList = isBlank(cql)
+    log.debug("getValidationRules:: Attempts to find validationRules by [offset: {}, limit: {}, cql: {}]",
+      offset, limit, cql);
+
+    boolean b = isBlank(cql);
```

Comment:
I think it's better to log JpaRepository methods somehow or just not log this value. Instead of name it `b`